pallets/btc-socket-queue/src/pallet/impls.rs

```diff
@@ -83,6 +83,86 @@ where
 		}
 	}
 
+	/// Try to verify PSBT inputs/outputs for RBF.
+	pub fn try_bump_fee_psbt_verification(
+		old_psbt: &Psbt,
+		new_psbt: &Psbt,
+	) -> Result<(), DispatchError> {
+		let old_psbt_inputs = &old_psbt.unsigned_tx.input;
+		let old_psbt_outputs = &old_psbt.unsigned_tx.output;
+
+		let new_psbt_inputs = &new_psbt.unsigned_tx.input;
+		let new_psbt_outputs = &new_psbt.unsigned_tx.output;
+
+		let current_round = T::RegistrationPool::get_current_round();
+		let system_vault = T::RegistrationPool::get_system_vault(current_round)
+			.ok_or(Error::<T>::SystemVaultDNE)?;
+
+		// output length check.
+		// the new output can possibly include/exclude an output for change.
+		if new_psbt_outputs.len().saturating_sub(old_psbt_outputs.len()) <= 1 {
```

Comment:
```suggestion
		if new_psbt_outputs.len().saturating_sub(old_psbt_outputs.len()) > 1 {
```
대체될 트랜잭션과 새 트랜잭션의 output 길이 차이가 1개까지 발생할 수 있다?

Reply:
오 감사합니다. 나이스 쳌